1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -7,9 +7,11 @@ export class BadInputError extends Error {
     }
 }
 
-export function fizzBuzz (input: any): string | Error{
-    if(typeof input != 'number') {
-        throw new BadInputError("input is not a valid number")
+export function fizzBuzz (input: number): string | Error{
```

Comment:
I don't believe you need to add `Error` as a return type of this function. Errors are thrown, not returned.

Reply:
Thanks, noted.